src/modules/notifications/notifications.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Controller, Delete, Req } from '@nestjs/common';
+import { NotificationsService } from './notifications.service';
+import { ApiBearerAuth, ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
+
+@ApiBearerAuth()
+@ApiTags('Notification')
+@Controller('notifications')
+export class NotificationsController {
+  constructor(private readonly notificationsService: NotificationsService) {}
+
+  @Delete('/clear')
+  @ApiResponse({ status: 200, description: 'Notifications cleared successfully' })
+  @ApiResponse({ status: 500, description: 'Internal server error' })
```

Comment:
Add more error response in this form with dtos as type:

```suggestion
@ApiOkResponse({type: successdto, description: ''})
@ApiUnauthorizedResponse({ type: , description: 'Unauthorized' })
  @ApiBadRequestResponse({ type: , description: 'Bad Request' })
  @ApiInternalServerErrorResponse({ type: , description: 'Internal Server Error' })
```